control/control.rnc

```diff
@@ -21,36 +21,57 @@ BOOLEAN =
     ## Historically we used config:type
     ## but it is unnecessarily long to type and read.
     ## Shorter variants are allowed.
-    attribute ( t | type | config:type ) { "boolean" },
+    # This could be expressed also as
+    # 'attribute ( t | type | config:type ) { "boolean" }'
+    # but libxml2-2.9.7 xmllint considers it incorrect schema :-(
```

Comment:
well, as we target SP3, what is libxml2 version there? If it is newer one, I think we need not to apply this ugly work-around.

Reply:
SLE15-SP3 doesn't exist in IBS yet and we don't know whether libxml2 will be updated to the Factory version (2.9.10) or not.
More important for me, I want to be able to check XML validity according to the latest schema even without running Tumbleweed.